src/main/java/co/refiere/dao/CampaignDao.java

```diff
@@ -68,6 +68,42 @@ public void deleteCampaign(Campaign campaign) {
             throw re;
         }
     }
+    
+    public List<Campaign> getReferencesCodesNull(int companyId) {
```

Comment:
Change it to use something more descriptive. Change `getReferencesCodesNull` to something like: `getReferres`